plugins/view-resources/src/components/EditDoc.svelte

```diff
@@ -104,7 +105,10 @@
         m.kind === ClassifierKind.MIXIN &&
         !ignoreMixins.has(m._id) &&
         (hierarchy.hasMixin(object, m._id) ||
-          (showAllMixins && hierarchy.isDerived(realObjectClass, hierarchy.getBaseClass(m._id))))
+          (showAllMixins &&
```

Comment:
Base class in our case is Person, not Employee, so we need to filter all mixins added to class extends our base class.

Reply:
Ouch, Employee is mixin for now, we need to check for child mixins, if document is already had it, and only in this case we should allow child mixins.

Reply:
Should be fixed now